static/js/iframe-common.js

```diff
@@ -135,6 +131,26 @@ export function generateIFrame(domain, answersExperienceFrame) {
   }, '#answers-frame');
 }
 
+function registerPopStateListener() {
+  let previousLocationHref = window.location.href
+
+  /**
+   * Reloads the iframe with a recalculated src URL.
+   * Does not reload the iframe if the URL has only changed its hash param. 
```

Comment:
so we'd still want regular query params to trigger a reload?

Reply:
We don't want to touch the preexisting behavior because we're not sure how it's being used or for what purpose it's there. This code was copy and pasted all at once from the AEB (answers experience builder created by Mcgin) and it doesn't seem possible to track that info down.